patroni/__main__.py

```diff
@@ -155,6 +155,10 @@ def reload_config(self, sighup: bool = False, local: Optional[bool] = False) ->
         except Exception:
             logger.exception('Failed to reload config_file=%s', self.config.config_file)
 
+    @property
```

Comment:
without going deep into the whole idea and the implementation - we have a dedicated abstract class for tags handling (see `tags.py`)

Reply:
@hughcapet, thank you, I was rebasing this patch from 2.1.7 version (the one we use in production) to current master, and some mistakes must have slipped through the cracks.
I will fix it.

Reply:
fixed, thank you for a feedback